BunnymodXT/modules/HwDLL.cpp

```diff
@@ -538,6 +540,8 @@ void HwDLL::Hook(const std::wstring& moduleName, void* moduleHandle, void* modul
 	}
 
 	#ifdef _WIN32
+		if (IsSecureClient())
+			MessageBox(NULL, "Encrypted client.dll, many of the BXT features will be unavailable. Please use one of the provided batch files to start the game.", NULL, MB_OK | MB_ICONWARNING);
```

Comment:
```suggestion
			MessageBox(NULL, "Encrypted client.dll, many of the BXT features will be unavailable. If using the GoldSrc Package, please run one of the provided batch files to start the game instead.", NULL, MB_OK | MB_ICONWARNING);
```
Makes it clearer for people who might not be using the package.